tests/arch/riscv/stack_unwind/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_TEST=y
```

Comment:
Given that this is also supported by ARM64 we should have a non-arch specific test for this.

Reply:
and x86 as well, that means we will need to have them printing stack traces in a similar way?

Reply:
Agreed, it is an annoyance that there is no standard way and that backtraces can be fragile based on optimizations and inlining.

But if there is a way that works for gcc and a way that works for llvm (possibly the same?), and there are a finite number of toolchains being supported, then that's already (scribbles on napkin) 99.9% of Zephyr's RISC-V users.

In any case, it isn't set in stone here.

This would be a great topic to raise with riscv.

Reply:
Added a test that should work for riscv, arm & x86, the only difference is the regex, since these platform output different things

> I found this useless for RISCV given that no standard actually exists for stack unwinding and each compiler is doing something different.

Since the Linux kernel has similar implementation for RISC-V, hopefully it is _generic enough_ to work on _most_ of the RISC-V SOCs.